drafts/25-xxx-macro-expansion.txt

```diff
@@ -182,26 +229,83 @@ EXAMPLE age04:
 This subsection describes the handling of line continuations
 within function-like macro invocations in free source form.
 
-lb01. An invocation of a function-like macro can be broken across
-      multiple lines. An invocation that begins with an open parenthesis
-      '(' continues until the matching closing parenthesis ')' is found
-      (outside of a comment), which may occur on a subsequent source
-      fragment line.
+lb01. [Straw poll 2, option 1] An invocation of a function-like macro
+      can be continued across multiple lines. An invocation that begins
+      with an open parenthesis '(' continues until the matching closing
+      parenthesis ')' is found (outside of a comment).
+
+lb01. [Straw poll 2, option 2] An invocation of a function-like macro
+      can be broken across multiple lines. An invocation that begins
+      with an open parenthesis '(' continues until the matching closing
+      parenthesis ')' is found (outside of a comment), which may occur
+      on a subsequent source fragment line.
 
 lb03. Comment lines within the argument list of an invocation of a
       function-like macro are discarded (section 2.1.4).
 
 lb05. A newline character appearing within the argument list is treated
       as a blank character, except where rule lb07 applies.
 
-lb07. Fortran-style continuation markers are permitted. An ampersand '&'
-      which is the last non-whitespace character (after the removal of
-      comments) preceding a newline character is removed, and all
-      subsequent characters up to and including the newline are removed.
-      If the first non-whitespace character on a continuation line is an
-      ampersand '&', all characters up to and including the leading
-      ampersand '&' are removed.
+lb07. [Straw poll 2, option 1] Fortran-style continuation markers are
+      required. An ampersand '&' which is the last non-whitespace
+      character (after the removal of comments) preceding a newline
+      character is removed, and all subsequent characters up to and
+      including the newline are removed. If the first non-whitespace
+      character on a continuation line is an ampersand '&', all
+      characters up to and including the leading ampersand '&' are
+      removed.
+
+lb07. [Straw poll 2, option 2] Fortran-style continuation markers are
+      permitted. An ampersand '&' which is the last non-whitespace
+      character (after the removal of comments) preceding a newline
+      character is removed, and all subsequent characters up to and
+      including the newline are removed. If the first non-whitespace
+      character on a continuation line is an ampersand '&', all
+      characters up to and including the leading ampersand '&' are
+      removed.
+
+[Straw poll 2, option 1]
+EXAMPLE lbe01:
```

Comment:
We now have two examples labelled `lbe01`, which could be verbally confusing in discussion.
I suggest we append a unique identifier to every such duplicate example, eg:

```suggestion
EXAMPLE lbe01-A:
```